### PR TITLE
Package database compat

### PR DESCRIPTION
**Pull request by Feiyang1**

### CODE REVIEW

File: packages/database/exp/index.node.ts [hunk 1]
```diff
@@ -17,61 +17,6 @@
...
-  startAt,
-  update
-} from '../src/exp/Reference_impl';
-export { increment, serverTimestamp } from '../src/exp/ServerValue';
-export {
-  runTransaction,
-  TransactionOptions,
-  TransactionResult
-} from '../src/exp/Transaction';
+export * from './api';
```
Comment (schmidt-sebastian) [inline packages/database/exp/index.node.ts:20]: Yay :)

File: packages/database/exp/api.ts [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  remove,
+  set,
+  setPriority,
+  setWithPriority,
+  startAfter,
+  startAt,
+  update,
+  ReferenceImpl as _ReferenceImpl,
+  QueryImpl as _QueryImpl,
+  child as _child
```
Comment (schmidt-sebastian) [inline packages/database/exp/api.ts:70]: `child` should be part of the public API. We can just export it as child (see go/rtdb-next)

File: packages/database/compat/index.ts [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+          ServerValue,
+          TEST_ACCESS
+        }
+      )
+      .setMultipleInstances(true)
+  );
+
+  instance.registerVersion(name, version);
+
+  if (isNodeSdk()) {
```
Comment (hsubox76) [inline packages/database/compat/index.ts:78]: I was wondering why this is here if a Node app should be using index.node.ts instead anyway, but I guess this is copied code and has been here for 4 years so this probably isn't the time to worry about it.

File: packages/database/compat/index.node.ts [hunk 4]
```diff
@@ -0,0 +1,153 @@
+/**
```
Comment (hsubox76) [inline packages/database/compat/index.node.ts:1]: Seems like index.node.ts and index.ts can share some code but don't know if it's worth worrying about that at this time.

**Review by hsubox76 — APPROVED** [review]

File: packages/database/exp/api.ts (hunk 2 above)
Comment (Feiyang1) [reply packages/database/exp/api.ts]: Thanks. Updated.

File: packages/database/compat/index.ts (hunk 3 above)
Comment (Feiyang1) [reply packages/database/compat/index.ts:78]: Good point. Removed.

File: packages/database/compat/package.json [hunk 5]
```diff
@@ -0,0 +1,13 @@
+{
+    "name": "@firebase/database-compat",
+    "version": "0.0.900",
+    "description": "The Cloud Storage component of the Firebase JS SDK.",
```
[the remark this answers is not in the document]
Comment (Feiyang1) [reply packages/database/compat/package.json]: Changed.

**Pull request merged**